deploy/install_local.step6.install_seqr.sh

```diff
@@ -87,6 +87,19 @@ python -u manage.py collectstatic --no-input
 python -u manage.py loaddata variant_tag_types
 python -u manage.py loaddata variant_searches
 
+# download and restore gene reference data
+for table_name in \
+    reference_data_humanphenotypeontology reference_data_mgi reference_data_omim reference_data_primateai reference_data_transcriptinfo \
```

Comment:
Would this work if you only dropped `reference_data_geneinfo`? Then you wouldn't need to maintain this list

Reply:
oh yeah